doc/rst/source/explain_symbols_ternary.rst_

```diff
@@ -14,138 +14,112 @@
     those values to suitable symbol sizes via the **-i** mechanism. The general
     input expectations are::
```

Comment:
Double colons mean that the following lines are codes, so the documentation is rendered like 
![image](https://user-images.githubusercontent.com/3974108/92652916-99b7ff00-f2bb-11ea-9d8a-9ba39826b7f2.png)

We should use a single colon here:
```suggestion
    input expectations are:
```